services/web/server/src/simcore_service_webserver/socketio/handlers.py

```diff
@@ -135,3 +146,22 @@ async def disconnect(sid: str, app: web.Application) -> None:
                 sid,
                 str(socketio_session),
             )
+
+@register_socketio_handler
+async def client_heartbeat(sid: str, _: Any, app: web.Application) -> None:
+    """JS client invokes this handler to signal its presence.
+
+    Each time this event is received the alive key's TTL is updated in
+    Redis. Once the key expires, resources will be garbage collected.
+
+    Arguments:
```

Comment:
Nice doc! We should adopt a convention on the format (sphinx, google, etc..) and start auto-generating doc ASAP

Reply:
Used the existing format, even though is not a standard, usually Python's docs have slightly different way of expressing arguments. There is also [PEP 257](https://www.python.org/dev/peps/pep-0257/). This is all I know.